### DISCUSSION TITLE
Idle timeout does not work.

### DISCUSSION THREAD

**Discussion by 7ombie** [helix-editor/helix#7452] [General]
### Summary

The `idle-timeout` configuration option appears to have no effect.

### Reproduction Steps

In your `config.toml` file, under `[editor]` do something like `idle-timeout = 10000`. Then reload the configuration file, reboot your LSP (maybe restart Helix for good measure). In any case, when you type `foo.` (when `foo` is assigned to some struct in the environment), the completions dialog opens instantly (as soon as you type the dot) to display every member of `foo`. The dialog should not appear unless you idle for 10,000 milliseconds (10 seconds).

### Helix log

n/a

### Platform

macOS Ventura 13.4.1

### Terminal Emulator

iTerm2 (Build 3.4.19)

### Helix Version

helix 23.05 (7f5940be)

**Comment by the-mikedavis** [comment]
That completion is caused by trigger characters rather than the idle timeout https://github.com/helix-editor/helix/blob/b33516fb16df93572ac4ddec1d4079282a2b4b5a/helix-term/src/commands.rs#L3315-L3323
